src/ResourceManager/DataLakeStore/Commands.DataLakeStore/Commands/RemoveAzureRmDataLakeStoreItemAcl.cs

```diff
@@ -47,40 +47,37 @@ public class RemoveAzureDataLakeStoreItemAcl : DataLakeStoreFileSystemCmdletBase
                 "Indicates that the ACL should be removed on the file with the specified ACL without prompting.")]
         public SwitchParameter Force { get; set; }
 
+        [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = false,
+            HelpMessage =
+                "Indicates a boolean response should be returned indicating the result of the delete operation."
+            )]
+        public SwitchParameter PassThru { get; set; }
+
         public override void ExecuteCmdlet()
         {
-            if (!Force.IsPresent)
-            {
-                ConfirmAction(
-                    Force.IsPresent,
-                    string.Format(Resources.RemovingDataLakeStoreItemAcl,
-                        Default ? Resources.DefaultAclWord : string.Empty, Path.OriginalPath),
-                    string.Format(Resources.RemoveDataLakeStoreItemAcl,
-                        Default ? Resources.DefaultAclWord : string.Empty, Path.OriginalPath),
-                    Path.OriginalPath,
-                    () =>
-                    {
-                        if (Default)
-                        {
-                            DataLakeStoreFileSystemClient.RemoveDefaultAcl(Path.TransformedPath, Account);
-                        }
-                        else
-                        {
-                            DataLakeStoreFileSystemClient.RemoveAcl(Path.TransformedPath, Account);
-                        }
-                    });
-            }
-            else
-            {
-                if (Default)
-                {
-                    DataLakeStoreFileSystemClient.RemoveDefaultAcl(Path.TransformedPath, Account);
-                }
-                else
+            ConfirmAction(
+                Force,
+                string.Format(Resources.RemovingDataLakeStoreItemAcl,
+                    Default ? Resources.DefaultAclWord : string.Empty, Path.OriginalPath),
+                string.Format(Resources.RemoveDataLakeStoreItemAcl,
+                    Default ? Resources.DefaultAclWord : string.Empty, Path.OriginalPath),
+                Path.OriginalPath,
+                () =>
                 {
-                    DataLakeStoreFileSystemClient.RemoveAcl(Path.TransformedPath, Account);
-                }
-            }
+                    if (Default)
+                    {
+                        DataLakeStoreFileSystemClient.RemoveDefaultAcl(Path.TransformedPath, Account);
+                    }
+                    else
+                    {
+                        DataLakeStoreFileSystemClient.RemoveAcl(Path.TransformedPath, Account);
+                    }
+
+                    if (PassThru)
+                    {
+                        WriteObject(true);
```

Comment:
@begoldsm if the user doesn't want output from the cmdlet, shouldn't the `bool` represent if the removal was successful or not, not just a hard-coded value?